.ci/integrationTestSelector.groovy

```diff
@@ -209,7 +209,7 @@ pipeline {
   post {
     cleanup {
       githubCheckNotify(currentBuild.currentResult == 'SUCCESS' ? 'SUCCESS' : 'FAILURE')
-      notifyBuildResult()
+      notifyBuildResult(prComment: false)
```

Comment:
GitHub PR comments from the parenstream rather than the test-selector job. It might change in the future to support multiple pipelines for the same PR, but for the time being let's centralise the notifications from the parenstreams only